package.json

```diff
@@ -14,7 +14,8 @@
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.3",
     "react-test-renderer": "^16.13.1",
-    "styled-components": "^5.1.1"
+    "styled-components": "^5.1.1",
+    "uuid": "^8.3.0"
```

Comment:
You add uuid, but never use it. Do you need it?

Reply:
No I don't. Added it because I thought about changing my key, but I didn't. Will delete this from the package.json. Thanks for the note!